scipy/optimize/_differentialevolution.py

```diff
@@ -181,6 +189,54 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     >>> result.x, result.fun
     (array([ 0.,  0.]), 4.4408920985006262e-16)
 
+
+    Next example demonstrates the parallelization capabilities.
+
+    >>>
+    from scipy.optimize import rosen
+    from scipy.optimize import differential_evolution as de
+    from scipy.misc import SPool
+    from scipy.misc import PPool
+    import time
+
+    def objfunc(params):
+        for it in xrange(1000000):
+            it**2
+        return rosen(params)
+
+    bounds = [(0,2), (0, 2), (0,2)]
+
+    start_time = time.time()
+    p = PPool()
+    result = de(objfunc, bounds, maxiter=10, polish=False, pool=p)
+    print("Parallel: %s seconds ---" % (time.time() - start_time))
+    print(result)
+
+    start_time = time.time()
+    result = de(objfunc, bounds, maxiter=10, polish=False)
+    print("Serial: %s seconds ---" % (time.time() - start_time))
+    print(result)
+    >>>
+    
+    Result in this case shows significant speedup:
```

Comment:
I am not sure if that it is possible with stochastic optimisations to compare results based on reaching some degree of convergence. It might happen that for some of the cases the nfev can be significantly higher then for other due to the probabilistic nature of the algorithm. The example shows that the parallelization can be significantly faster to make a certain number of nfev. In this case the optimization procedure was limited by the number of objective function evaluations.